README.md

```diff
@@ -25,15 +26,19 @@ For more information, see the [Intel TDX overview](https://www.intel.com/content
 This tech preview of Intel TDX on Ubuntu 24.04 provides base host OS, guest OS, and remote attestation functionalities.
 Follow these instructions to setup the Intel TDX host, create a TD, boot the TD, and attest the integrity of the TD's execution environment.
 
-The setup can be customized by editing the global configuration file: `setup-tdx-config`
```

Comment:
Why do you want to remove this line ?

Reply:
I moved it to 4.2.2 which makes more sense in the sequence of setting up TDX.

Reply:
I agree, but you should mention it again in the section 5.

Reply:
I think we should keep it as it is before this MR, the config file will cover (now and in the future) all setup actions; i would like to have it at the beginning of the README

Reply:
How about a compromise?  Put it back like you requested, but also add it here with a little description as I proposed.  If it's at the beginning alone, I'm afraid it can be easily missed.

Reply:
Ok